kuma/static/js/components/banners/utils/banners-event-util.js

```diff
@@ -0,0 +1,74 @@
+(function() {
+    'use strict';
+    var mdnBannerEvents = {
+        /**
+         * Attach common event handlers for the CTA banner.
+         * @param {Object} banner - The banner element
+         */
+        attachEvents: function(banner) {
+            this.handleCollapseEvent(banner);
+            this.handleCloseEvent(banner);
+            this.handleExternalLinkEvent(banner);
+        },
+        /**
+         * Handles click events on the close(x) icon
+         * @param {Object} banner - The banner element
+         */
+        handleCloseEvent: function(banner) {
+            var closeButton = banner.querySelector('.mdn-cta-close');
+
+            if (!closeButton) {
+                return;
+            }
+
+            closeButton.addEventListener('click', function() {
+                banner.classList.add('hidden');
+                window.mdnBannersStateUtil.setBannerInactive(
+                    banner.dataset['banner']
+                );
+            });
+        },
+        /**
+         * Handles click events on the collapse icon
+         * @param {Object} banner - The banner element
+         */
+        handleCollapseEvent: function(banner) {
+            var collapseButton = banner.querySelector('.mdn-cta-collapse');
+
+            if (!collapseButton) {
+                return;
+            }
+
+            collapseButton.addEventListener('click', function() {
+                banner.classList.remove('expanded');
+                banner.classList.add('is-collapsed');
+            });
+        },
+        /**
+         * Handles click events on elements(typically a button) with
```

Comment:
If it is a button, then it won't have an href.... As written, I think it only works for links, right?

Reply:
My bad, I did not mean a button in the literal sense. Will update the comment. It will always be an anchor element as it takes you somewhere, as opposed to a button which typically does something.